index.js

```diff
@@ -1,11 +1,44 @@
 'use strict';
 const fs = require('fs');
+const os = require('os');
 const path = require('path');
 const childProcess = require('child_process');
 const runApplescript = require('run-applescript');
 const pify = require('pify');
 const rimraf = require('rimraf');
-const xdgTrashdir = require('xdg-trashdir');
```

Comment:
on second thought I think I'll pass on that :-) 
It seems that to properly extend that module I'd have to break the signature of the `export`...
(otherwise it would get ugly)